eclair-core/src/main/scala/fr/acinq/eclair/json/JsonSerializers.scala

```diff
@@ -441,9 +441,10 @@ object InvoiceSerializer extends MinimalSerializer({
         MilliSatoshiSerializer +
         CltvExpiryDeltaSerializer
     ))
+    import fr.acinq.eclair.json.JsonSerializers.formats
     val fieldList = List(
       JField("prefix", JString(p.prefix)),
-      JField("timestamp", JLong(p.createdAt.toLong)),
+      JField("timestamp", Extraction.decompose(p.createdAt)),
```

Comment:
Why are changes to this file needed? They're less readable than the previous version, and the `iso` field is unnecessary, a unix timestamp is enough?

Reply:
Fair enough, why not!